File: reservation-service/src/main/java/rs/raf/reservationservice/domain/Reservation.java
package rs.raf.reservationservice.domain;

import javax.persistence.*;
import lombok.*;

import java.util.Date;

@Entity
@Table(name = "reservation")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Reservation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "client_user_id")
    private Long client_user_id;
    @ManyToOne
    private VehicleInFirm vehicle_in_firm;
    @Column(name = "start_date")
    private Date start_date;
    @Column(name = "end_date")
    private Date end_date;
}

File: ReservationServiceClient/model/reservationservice/base/Reservation.cs

public class Reservation {
    public long id { get; set; }
    public long client_user_id { get; set; }
    public long vehicle_in_firm_id { get; set; }
    public DateTime start_date { get; set; }
    public DateTime end_date { get; set; }
    public Reservation() {
    }

    public Reservation(long id, long client_user_id, long vehicle_in_firm_id, DateTime start_date, DateTime end_date) {
        this.id = id;
        this.client_user_id = client_user_id;
        this.vehicle_in_firm_id = vehicle_in_firm_id;
        this.start_date = start_date;
        this.end_date = end_date;
    }

    


}
